profanity_word_phrase_filter/README.md

```diff
@@ -1,19 +1,20 @@
 ## Custom profanity filter
 
-Most online platforms contain inappropriate language. Parent's want to be certain
+Most online platforms contain inappropriate language. Parents want to be certain
 that Story squad (aka scribble stadium) is offering a safe environment for their
-kiddos. In compliance to the children's Online Protection rule (COPPA)
+kiddos in compliance with the Children's Online Privacy Protection Rule (COPPA).
 https://www.ftc.gov/enforcement/rules/rulemaking-regulatory-reform-proceedings/childrens-online-privacy-protection-rule
 
-It's important for the submitted stories to be flagged as inappropriate if they 
-contain any profanity. The stories will still be viewed by a human eye to make
-sure that the flagging is warranted. 
-The previous cohort worked on using the available libraries like profanity-filter
-and profanity_Check but they were not sufficient in flagging all words and newer
+It is important for submitted stories to be flagged as inappropriate if they 
+contain any profanity.The stories will still be reviewed manually by a person to make
+sure flagging, by the system, is warranted. 
+The previous cohort worked on using available libraries like profanity-filter
```

Comment:
Delete mentions of previous cohorts and timelines. Say something like "Libraries like profanity-filter and profanity_Check do not sufficiently flag all profane words and phrases. Either a custom filter needs to be developed or another library found.

Reply:
Have rewritten the readme. Let me know if there is more changes that should be made. Also lets follow the advise by Ash to request for changes or approve request so we can make progress.